### PR TITLE
Update Welcome Page

### PR DESCRIPTION
**Pull request by gryter**
Reference: Issue #2857 

Welcome page updated.  Details outlined in Issue.

### CODE REVIEW

File: src/app/help/en/welcome.md [hunk 1]
```diff
@@ -164,24 +163,24 @@ Once you follow someone, all of their posts will show up in your "Feed" on the h
...
-STEEM, Steem Power and Steem Dollars are the three forms of digital currency used by the Steem Blockchain.
+STEEM, Steem Power and Steem Dollars are the three forms of digital currency tokens used by the Steem Blockchain.
 
 More information on the three types of tokens can be found in the [Steemit FAQ](https://steemit.com/faq.html#What_is_the_difference_between_STEEM__STEEM_Power__and_Steem_Dollars).
 
 <a href="#Table_of_Contents">^</a>
 ### <span id="Curation">Curation</span>
 
-Up to 25% of the reward for posts goes to the people who voted on it. These people are called curators.
+Up to 25% of the reward for each publication goes to the people who voted on it. These people are called Curators.
```
Comment (Smittyvb) [inline src/app/help/en/welcome.md]: Steemit uses the word "post" everywhere, not "publication". Also, why is "curator" capitalized?

File: src/app/help/en/welcome.md [hunk 2]
```diff
@@ -190,24 +189,24 @@ The more Steem Power you have in your account, the more your upvotes will be wor
...
-A reputation score is one way Steemit measures the amount of value you have brought to the community.
+A reputation score is one way that Steemit measures the amount of value you have brought to the community.
 
-The higher the number, the more weighted votes an account has earned.
+The higher the reputation, the more upvotes the account received.
 
 All new users start at 25.
 
-Your reputation will go up as you earn upvotes for your posts and comments, but it can come down if they are flagged.
+Your reputation will go up as you earn upvotes for your posts and comments, but it can come down if they are downvoted.
```
Comment (Smittyvb) [inline src/app/help/en/welcome.md]: The UI refers to downvotes as flags.

File: src/app/help/en/welcome.md [hunk 3]
```diff
@@ -221,87 +220,88 @@ You can also exchange your STEEM and SBD for bitcoin on a third-party exchange s
...
+The community is looking for you to create engaging content. 
 
-Plagiarizing, that is posting someone else's work as if it were your own, is very frowned upon by the Steemit community.
+Plagiarising, which is posting someone else's material as if it were your own, is very frowned upon by the Steem community.
 
-If you are using anyone else's material as part of your posts (including images) - please cite your sources.
+If you are using anyone else's material as part of your posts (including images), even if it is free to use, please cite your sources. You should make sure that you are not violating any copyright laws if you are using someone else's material/images. 
 
-Also, make sure that you are not violating any copyright laws if you are using someone else's material/images. Limited, sourced material sharing is OK under fair use and fair dealing doctrines.
+Remember, your posts should have original content in them. Quotes should not exceed more than half of a publication. Include your own thoughts, photographs, art, and ideas.
```
Comment (Smittyvb) [inline src/app/help/en/welcome.md]: Steemit uses the word "post" everywhere, not "publication".

File: src/app/help/en/welcome.md [hunk 4]
```diff
@@ -310,9 +310,7 @@ New Member Support Community is a group of people dedicated to helping new users
 <a href="#Table_of_Contents">^</a>
 ## <span id="Third_Party_References">Third Party References</span>
 
-Peerhub, BlockTrades, Bittrex, Steem Chat, Steemit Help, New Member Support Community, and Discord Chat, as well as the tools listed under "Other Resources" are third party applications/services, and are not owned or maintained by Steemit, Inc. Their listing here does not constitute and endorsement or recommendation on behalf of Steemit, Inc.
-
-All of the links in the "Helpful Posts from Steemit Users" section were created by our users and do not necessarily represent the views of Steemit, Inc. or its management.
```
Comment (Smittyvb) [inline src/app/help/en/welcome.md]: Why was this line removed?

**Comment by Smittyvb** [comment]
It looks like you made a few changes for the sake of improving auto-translation. Instead of doing that, we should just have a full translated copy of this document for other languages, even if that's just a modified version of this document through Google Translate.
<sub>Also, CodeBuild always fails for outside users, you didn't do anything to cause build errors.</sub>

**Comment by Smittyvb** [comment]
The list of posts should all be relative (use ``/steem-help/@timcliff/the-ultimate-guide-of-tips-for-new-steem-user`` not ``https://steemit.com/steem-help/@timcliff/the-ultimate-guide-of-tips-for-new-steem-user``). That way condenser instances other than Steemit.com will get internal links.

File: src/app/help/en/welcome.md [hunk 5]
```diff
@@ -39,27 +37,24 @@ Below that is a list of recommended users to follow, a collection of other resou
...
-Unlike centralized web services, **the Steem Blockchain has no account password recovery**.
+### <span id="Backup_your_password">1. Backup Your Password</span>
 
-You are entirely responsible for keeping your password, and keeping it secure.
+Unlike centralized web services, **it is not possible to recover lost passwords on the Steem blockchain**.
 
-Save your master key and keep it somewhere safe.
+You are entirely responsible for generating your password, saving it and keeping it secure.
 
-It is strongly recommended that you store an offline copy of your password somewhere safe in case of a hard drive failure or other calamity. Consider digital offline storage, such as a flash drive or burned CD, as well as printed paper. Use a safe deposit box for best redundancy.
```
[the remark this answers is not in the document]
Comment (gryter) [reply src/app/help/en/welcome.md]: Passed this through a friend who is a senior security consultant. Flash drives and burned CDs fall under digital offline storage. Secure cloud storage was recommended as an accessible alternative for mobile users, which encompasses a large portion of Steemit users. Safe deposit boxes are only accessible to a small user base and confusing.

File: src/app/help/en/welcome.md [hunk 6]
```diff
@@ -147,13 +146,13 @@ To comment on a post, or reply to an existing comment, click on the "Reply" link
 
 To create a post, click on the "Post" link in the upper right corner.
 
-Posts have three main parts: Title, Content, Tags.
+Posts have three main parts: Title, Content, and Tags.
 
-You will want to make your title attention grabbing, and relevant to your content. The first image that you use will be the thumbnail image.
```
[the remark this answers is not in the document]
Comment (gryter) [reply src/app/help/en/welcome.md]: 'Attention grabbing' was not translating at all and, disregarding translation, was confusing to users whose use of English is their second language.

File: src/app/help/en/welcome.md (hunk 2 above)
Comment (gryter) [reply src/app/help/en/welcome.md]: The word 'flag' was confusing to users; particularly to those who were translating.  The thought process behind changing it to 'downvoted' was to clarify it as being opposite of upvoting, which is mentioned in the same sentence.  It is meant to make the sentence easier to understand for users who are not yet familiar of how upvotes and flags look like on a post.

File: src/app/help/en/welcome.md (hunk 4 above)
Comment (gryter) [reply src/app/help/en/welcome.md]: It should be reinserted.  It was originally removed during editing because the list of posts had posts by Steemit Inc personnel.

**Comment by gryter** [comment]
Would the contributor team be open to translations being prepared and submitted in the stead of some of the equalizing segments of text? Numerous fluently-bilingual users are waiting for the go-ahead to translate. 

Will update accordingly and resubmit PR. 

I added reasoning/thought-process above.

File: src/app/help/en/welcome.md (hunk 2 above)
Comment (Gandalf-the-Grey) [reply src/app/help/en/welcome.md]: Still it's not consistent with current UI, so IMO it should be changed *only if* flag will be renamed to downvote.

File: src/app/help/en/welcome.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Gandalf-the-Grey) [reply src/app/help/en/welcome.md]: This paragraph is about offline storage. Cloud based storage is out of scope. Not to mention that if your e-mail will be compromised it also means that your secure cloud-based storage is no longer secure. Certain set of celebrities can tell you the difference between "secure cloud-based storage" and a safe deposit box. I cannot agree that they are accessible to a small user base. It's a common service available in banks and post offices. Also, that's just an example of offline storage for redundancy.
Real life scenario: a steemian with his house burned to the ground.

File: src/app/help/en/welcome.md (hunk 1 above)
Comment (Gandalf-the-Grey) [reply src/app/help/en/welcome.md]: digital currency => digital currency tokens 
IMHO redundant and confusing

File: src/app/help/en/welcome.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (gryter) [reply src/app/help/en/welcome.md]: I ran your concerns past the consultant. What they're suggesting is secure (private) cloud vs public cloud or cloud designed for basic storage. There is a major security report for the crypto industry coming out this summer I'm told. They're not recommending banks because its not an option in most parts of the world and safety deposit boxes can be seized. Relating the info I got on that.

File: src/app/help/en/welcome.md (hunk 2 above)
Comment (TimCliff) [reply src/app/help/en/welcome.md]: A happy medium may be to go with the verbiage used above:
_downvote (flag)_

File: src/app/help/en/welcome.md (hunk 3 above)
Comment (TimCliff) [reply src/app/help/en/welcome.md]: I would not remove this part, as it gives a fairly clear message based on legally defined terms:
_Limited, sourced material sharing is OK under fair use and fair dealing doctrines._

Comment (TimCliff) [reply src/app/help/en/welcome.md]: _Quotes should not exceed more than half of a publication._

That seems subjective. What if someone wants to publish a quotes list?

File: src/app/help/en/welcome.md [hunk 7]
```diff
@@ -5,26 +5,24 @@ This page is full of information to help you learn about the platform and become
 
 Included on the page is a "To Do List" with recommended steps to get started with your account, and a "Quick Start Guide" with information on how the platform works. 
 
-Below that is a section of "Helpful Posts from Steemit Users", which contains a collection of posts from users in the community that are helpful for new users getting started.
```
Comment (TimCliff) [inline src/app/help/en/welcome.md]: This should be added back.

File: src/app/help/en/welcome.md (hunk 3 above)
Comment (gryter) [reply src/app/help/en/welcome.md]: They'll get flagged for copy/paste without original content in the post.  It's an ongoing issue.

Comment (TimCliff) [reply src/app/help/en/welcome.md]: I understand that non-original copy/paste content will get downvoted. That seems valid and we should communicate it.  I don’t think that the “half of the post should be non copied” rule is the right way to say it though.

Comment (gryter) [reply src/app/help/en/welcome.md]: What if we go with "Quotes should not compose an entire publication."

Comment (TimCliff) [reply src/app/help/en/welcome.md]: Sounds ok to me. I’d also suggest adding “fair use and fair dealing doctrines“ back in too though.

File: src/app/help/en/welcome.md (hunk 2 above)
Comment (humphreybear) [reply src/app/help/en/welcome.md]: Depending on the language to which it is being translated into, there is no way possible to literally translate some terms. The word flag may in this case be used as a term for downvoting however when translated into some other language it won't make any sense as many a language group already have their terms for many such actions.

The one that always comes to mind is the term "Like" on one of the other well known social networks out there. In many a country located in the Balkans, just as one example that I am fully familiar with, they use the term "lajkanje" which can't be found in any dictionary, yet it is understood by one and all what it means. Hence I would have to agree with a logical approach which in this case would be as TimCliff suggested "downvote (flag)".

Same applies to the word Curation, the literal translation of this word would make absolutely no sense in the language used in some of the countries of the Balkan region. Hence in the translation it would have to use wording that describes the action with the English word in brackets. So as to ensure that there is nothing lost in translation and that the original English version takes precedence in such situations.

This is not a project that any "official translator" or "online automated translator" can accomplish for any one language. 

To back this statement up, my wife who's first language is Croatian and English is her second language who has gone ahead and taken the initiative to do such a translation of the front page came across this situation on numerous points and when I compared her correct translation as per the grammar of that language to what the "online translator" gave me, it was way different. Adding to the need for a logical approach to translating pages, and not some automated or official literal translation which shall not make any sense to people reading it. 

ref: https://steemit.com/steemit/@sylviamiller/translation-of-the-welcome-page-croatian-hrvatski-finished

anyone is more than welcome to go and use an online translator and compare the differences. 

Jack

Comment (gryter) [reply src/app/help/en/welcome.md]: That was one of the main points behind getting it optimized and to pave the way for translation.

**Comment by X40L1N** [comment]
Coming together nicely if you ask me. Was quite the clustercluck and misleading. 

*pats BRO on the back*

**Comment by plink01001** [comment]
@gryter thank you for your contribution! Great job on the change.

There are still a small handful of comments that appear to be unaddressed though. Can you please review all of the conversations and ensure the necessary changes have been made? If you are unsure about how to proceed on any of them (due to conflicting advise) just ask me and I'll let you know. 

Ping me once you have made all the updates based on the feedback and I'll do another round of review.

**Comment by gryter** [comment]
The requested changes are now made.

**Comment by X40L1N** [comment]
Gettin it done buddy, thanks for your commitment.

**Comment by plink01001** [comment]
@gryter looking good!

Can you please make a few minor updates:

1. Users who sign up via the faucet will not be "generating" a password.
_You are entirely responsible for generating your password, saving it and keeping it secure._
Please change to:
_You are entirely responsible for saving your password, backing it up, and keeping it secure._

2. A lot of users may not know what "digital offline storage" is. Let's clarify.
_Consider digital offline storage, secure private cloud-based storage..._
Please change to:
_Consider digital offline storage (such as a flash drive or burned CD), secure private cloud-based storage..._

3. This change is not made consistently across the page.
`<span id="Upvotes">Votes</span>`
Please change "Upvotes" to "Votes" in all places.

4. Inc should have a period.
_Ned Scott, CEO and Co-Founder of Steemit, Inc_
Please change to:
_Ned Scott, CEO and Co-Founder of Steemit, Inc._

**Comment by gryter** [comment]
Apologies on late changes.  Now made.

**Comment by plink01001** [comment]
@gryter thanks! 

It is very close, but I still see a few things that need to be updated.

For item 1, can you please change the wording to:
_You are entirely responsible for saving your password, backing it up, and keeping it secure._

For item 3, there are a few places that "upvote" was changed to "vote" which should have been left as "upvote". Can you please change these back:
- _Upvotes are Steemit's way of saying you like someone's post or comment._
- _The higher the reputation, the more upvotes the account received._
- _Your reputation will go up as you earn upvotes for your posts and comments, but it can come down if they are downvoted (flagged)._

**Comment by originated** [comment]
@plink01001 Please review https://github.com/steemit/condenser/pull/2946 (This has your last requested changes plus original updates).

**Comment by roadscape** [comment]
I've moved the commit history to local branch @ #2946, please confirm it reflects the requested changes. Thanks all!
